src/test/shared/awsFiletypes.test.ts

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as assert from 'assert'
+import * as path from 'path'
+import globals from '../../shared/extensionGlobals'
+import * as sysutil from '../../shared/systemUtilities'
+import * as testUtil from '../testUtil'
+import * as workspaceUtils from '../../shared/utilities/workspaceUtils'
+import { toArrayAsync } from '../../shared/utilities/collectionUtils'
+import { waitUntil } from '../../shared/utilities/timeoutUtils'
+
+describe('awsFiletypes', function () {
+    let awsConfigUri: vscode.Uri | undefined
+    let cfnUri: vscode.Uri | undefined
+
+    beforeEach(async function () {
+        globals.telemetry.clearRecords()
+        testUtil.closeAllEditors()
+
+        // Create a dummy file in ~/.aws on the system.
+        // Note: We consider _any_ file in ~/.aws to be an "AWS config" file,
+        // so this will trigger "file_editAwsFile" telemetry.
+        const awsConfigFile = path.join(sysutil.SystemUtilities.getHomeDirectory(), '.aws/test_awstoolkit')
```

Comment:
This writes a file to `~/.aws`. Not ideal, but not destructive, only affects Toolkit developers, and can be removed once we have a way to override the "home directory" in tests.